packages/common/src/types/certificate.ts

```diff
@@ -445,6 +471,26 @@ export interface ValueMedicalInvestigationList extends Value {
   list: ValueMedicalInvestigation[]
 }
 
+export interface ValueViewText extends Value {
+  type: CertificateDataValueType.VIEW_TEXT
+  text: string
+}
+
+export interface ValueViewList extends Value {
+  type: CertificateDataValueType.LIST
```

Comment:
Should we add the VIEW prefix to list, similar to VIEW_TEXT?

Reply:
Definitely